modules/platform/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/web/common/requestcontroller/filter/NuxeoRequestControllerFilter.java

```diff
@@ -138,9 +138,18 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
             }
             chain.doFilter(request, response);
         } catch (IOException | ServletException | RuntimeException e) {
+            if (!response.isCommitted()) {
+                response.resetBuffer();
```

Comment:
this change is not directly related, right?

Reply:
In order for the exception filter to be able to send a custom JSON with the exception, the existing output has to be cleared. And it has to be done here where we still have a buffered response that can be wiped out, before we give back control to the higher filter that doesn't have any buffering.